vaadin-renderer-flow-parent/vaadin-renderer-flow/src/main/java/com/vaadin/flow/data/renderer/NativeButtonRenderer.java

```diff
@@ -88,6 +88,10 @@ public NativeButtonRenderer(String label,
      */
     public NativeButtonRenderer(ValueProvider<SOURCE, String> labelProvider) {
```

Comment:
Unrelated: Why does this even exist in our public API? Maybe worth a discussion if we want to deprecate / remove it.